inventory_price_check/models/product_template.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    price_discrepancy_threshold_type = fields.Selection(
+        [("percentage", "Percentage"), ("fixed", "Fixed Value"), ("ignore", "Ignore")]
+    )
+    price_discrepancy_threshold_value = fields.Float(
+        help="Threshold value for price discrepancy warnings."
+    )
```

Comment:
Can we make it like this:
```suggestion
    price_variance_threshold_percent = fields.Float(help="Maximum variance (in percent) allowable between the product's standard price and purchase receipt unit price.")
    price_variance_threshold_amount = fields.Float(help="Maximum allowable variance (in monetary amount, based on company currency) between the product's standard price and the purchase receipt unit price.")
```

The idea is to allow both types of thresholds. Hitting either one should trigger an error.